### ISSUE TITLE
pip install dev does not work

### ISSUE THREAD

**Issue by thomasyu888** [Sage-Bionetworks/synapsePythonClient#311]
```
sudo pip install git+https://github.com/Sage-Bionetworks/synapsePythonClient.git@develop
Downloading/unpacking git+https://github.com/Sage-Bionetworks/synapsePythonClient.git@develop
  Cloning https://github.com/Sage-Bionetworks/synapsePythonClient.git (to develop) to /tmp/pip-ouux3d-build
  Running setup.py (path:/tmp/pip-ouux3d-build/setup.py) egg_info for package from git+https://github.com/Sage-Bionetworks/synapsePythonClient.git@develop

Requirement already satisfied (use --upgrade to upgrade): requests>=1.2 in /usr/lib/python2.7/dist-packages (from synapseclient==1.5.2.dev1)
Requirement already satisfied (use --upgrade to upgrade): six in /usr/lib/python2.7/dist-packages (from synapseclient==1.5.2.dev1)
Downloading/unpacking future (from synapseclient==1.5.2.dev1)
  Downloading future-0.15.2.tar.gz (1.6MB): 1.6MB downloaded
  Running setup.py (path:/tmp/pip_build_root/future/setup.py) egg_info for package future

    warning: no files found matching '*.au' under directory 'tests'
    warning: no files found matching '*.gif' under directory 'tests'
    warning: no files found matching '*.txt' under directory 'tests'
Downloading/unpacking backports.csv (from synapseclient==1.5.2.dev1)
  Downloading backports.csv-1.0.1-py2.py3-none-any.whl
Installing collected packages: future, backports.csv, synapseclient
  Running setup.py install for future

    warning: no files found matching '*.au' under directory 'tests'
    warning: no files found matching '*.gif' under directory 'tests'
    warning: no files found matching '*.txt' under directory 'tests'
    Installing pasteurize script to /usr/local/bin
    Installing futurize script to /usr/local/bin
  Running setup.py install for synapseclient

    Installing synapse script to /usr/local/bin
Successfully installed future backports.csv synapseclient
Cleaning up...
```

I suspect it is mainly because of this line : `Running setup.py install for synapseclient`.  Even when we clone the develop, if you `python setup.py install` it will not install the dev branch.  You must do `python setup.py develop` to install the dev branch.

**Comment by thomasyu888** [comment]
This will be fixed in the synapseutils batch.  Setup.py will be revised

**Comment by mcolic** [comment]
How do I install synapseutils? Is it a separate module, because when I install syanpseclient and try to import synapseutils it says module not found? Any help or advice would be appreciated.

**Comment by kdaily** [comment]
@mcolic it is available in the new release [1.6.1](https://github.com/Sage-Bionetworks/synapsePythonClient/releases/tag/v1.6.1) and now on the master branch here.

**Comment by mcolic** [comment]
Great thanks, I got it work :) @kdaily
